docs/building-blocks.md

```diff
@@ -157,26 +157,26 @@ The resulting implementation would look something like this:
 
 ```ruby
 SparkleFormation.dynamic(:elb) do |_name, _config={}|
-  resources("#{_name}_elb".to_sym) do
-    type 'AWS::ElasticLoadBalancing::LoadBalancer'
-    properties do
-      availability_zones._set('Fn::GetAZs', '')
-      listeners _array(
-        -> {
-          load_balancer_port _config[:load_balancer_port] || '80'
-          protocol _config[:protocol] || 'HTTP'
-          instance_port _config[:instance_port] || '80'
-          instance_protocol _config[:instance_protocol] || 'HTTP'
-        }
-      )
-      health_check do
-        target _config[:target] || 'HTTP:80/'
-        healthy_threshold _config[:healthy_threshold] || '3'
-        unhealthy_threshold _config[:unhealthy_threshold] || '3'
-        interval _config[:interval] || '5'
-        timeout _config[:timeout] || '15'
+    resources("#{_name}_elb".to_sym) do
```

Comment:
It's a small thing, but can you remove the extra space at the front of this block?